### PR TITLE
AC-4034: Setup a deploy pipeline in aws code-pipeline to deploy api server

### PR DESCRIPTION
**Pull request by shanbady**
Creating a preliminary PR for the deployment setup for the api module. instructions to follow

### CODE REVIEW

**Comment by mc-nathan** [comment]
.DS_store files should not be in the repo

**Comment by shanbady** [comment]
it just occurred to me that the travis build script tags all built images as :latest. This means that builds deployed from a master branch have the potential of taking precedence over images built from a particular branch. fixing this now by making the image tag be the commit hash. will update

**Comment by shanbady** [comment]
@mc-nathan I think i've addressed all the outstanding issues

File: README.md [hunk 1]
```diff
@@ -1,19 +1,6 @@
...
-
-	git clone https://github.com/masschallenge/impact-api.git
-	git clone https://github.com/masschallenge/impact-api-deploy
-	mv impact-api-deploy/ansible impact-api
-	cd impact-api
-
-### then execute a build
-
-	ansible-container build
+Travis is pre-configured to deploy to the impact api production instance anytime something is merged into master. All thje settings and secret keys are configured in travis via environment variables.
```
Comment (pbx) [inline README.md]: Capitalize "API"

File: README.md [hunk 2]
```diff
@@ -1,19 +1,6 @@
 # impact-api-deploy
 Contains the build and deploy components of the api server
```
Comment (pbx) [inline README.md:2]: Capitalize "API"

Comment (shanbady) [reply README.md:2]: done

**Review by pbx — APPROVED** [review]

**Pull request merged**